cmd/cloud/organization.go

```diff
@@ -54,6 +60,24 @@ func newOrganizationSwitchCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+func newOrganizationExportAuditLogs(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "export-audit-logs [organization_name] --output-file [output_file.json]",
+		Aliases: []string{"eal"},
+		Short:   "Export your organization audit logs. Requires being an organization owner.",
+		Long:    "Export your organization audit logs. Requires being an organization owner.",
+		Args:    cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return organizationExportAuditLogs(cmd, out, args)
+		},
+	}
+	cmd.PersistentFlags().StringVarP(&auditLogsOutputFilePath, "output-file", "o", "", "Path to a file for storing exported audit logs")
+	cmd.PersistentFlags().IntVarP(
```

Comment:
Does the endpoint to limit the logcount...if yes, can we add that ability to be able to pull only last `n` logs

Reply:
The endpoint currently only allows to select a time period in `days`.